src/openstudio_app/OpenStudioApp.cpp

```diff
@@ -1399,6 +1371,29 @@ void OpenStudioApp::loadLibrary() {
   }
 }
 
+void OpenStudioApp::loadExampleModel() {
+
+  bool wasQuitOnLastWindowClosed = this->quitOnLastWindowClosed();
+  this->setQuitOnLastWindowClosed(false);
+
+  if (m_osDocument) {
+    if (!closeDocument()) {
+      this->setQuitOnLastWindowClosed(wasQuitOnLastWindowClosed);
+      return;
+    }
+    processEvents();
+  }
+
+  auto model = openstudio::model::exampleModel();
+  m_osDocument = std::shared_ptr<OSDocument>(new OSDocument(componentLibrary(), resourcesPath(), model, QString(), false, startTabIndex()));
+
+  connectOSDocumentSignals();
```

Comment:
This looks like what is happening in `newModel` which is what I would copy.  I think the main difference between calling `loadExampleModel` and `newModel` repeatedly is that the example model has objects with connections.  Maybe `closeDocument` is missing something?